diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/base.mk cm11/build/target/product/base.mk
--- cm11-backup/build/target/product/base.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/base.mk	2014-11-14 05:13:31.492896560 +0100
@@ -25,7 +25,6 @@
     applypatch \
     blkid \
     bmgr \
-    bugreport \
     content \
     dhcpcd \
     dhcpcd-run-hooks \
diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/core.mk cm11/build/target/product/core.mk
--- cm11-backup/build/target/product/core.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/core.mk	2014-11-14 00:35:28.165064587 +0100
@@ -20,14 +20,11 @@
 # core_minimal.mk instead.
 
 PRODUCT_PACKAGES += \
-    BasicDreams \
-    Browser \
     Contacts \
     DocumentsUI \
     DownloadProviderUi \
     ExternalStorageProvider \
     KeyChain \
-    PicoTts \
     PacProcessor \
     ProxyHandler \
     SharedStorageBackup \
diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/core_base.mk cm11/build/target/product/core_base.mk
--- cm11-backup/build/target/product/core_base.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/core_base.mk	2014-11-14 00:35:50.781063003 +0100
@@ -30,7 +30,6 @@
     libgabi++ \
     libkeystore \
     libmdnssd \
-    libnfc_ndef \
     libportable \
     libpowermanager \
     libspeexresampler \
diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/full_base.mk cm11/build/target/product/full_base.mk
--- cm11-backup/build/target/product/full_base.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/full_base.mk	2014-11-14 05:16:44.877883021 +0100
@@ -34,10 +34,7 @@
 PRODUCT_AAPT_CONFIG := normal hdpi xhdpi xxhdpi
 
 # Get some sounds
-$(call inherit-product-if-exists, frameworks/base/data/sounds/AllAudio.mk)
-
-# Get the TTS language packs
-$(call inherit-product-if-exists, external/svox/pico/lang/all_pico_languages.mk)
+$(call inherit-product-if-exists, frameworks/base/data/sounds/OriginalAudio.mk)
 
 # Get a list of languages.
 $(call inherit-product, $(SRC_TARGET_DIR)/product/locales_full.mk)
diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/full_base_telephony.mk cm11/build/target/product/full_base_telephony.mk
--- cm11-backup/build/target/product/full_base_telephony.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/full_base_telephony.mk	2014-11-14 00:37:11.492057353 +0100
@@ -19,9 +19,6 @@
 # build quite specifically for the emulator, and might not be
 # entirely appropriate to inherit from for on-device configurations.
 
-PRODUCT_PACKAGES := \
-    VoiceDialer
-
 PRODUCT_PROPERTY_OVERRIDES := \
     keyguard.no_require_sim=true
 
diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/generic_no_telephony.mk cm11/build/target/product/generic_no_telephony.mk
--- cm11-backup/build/target/product/generic_no_telephony.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/generic_no_telephony.mk	2014-11-14 05:17:36.676879394 +0100
@@ -20,21 +20,15 @@
 PRODUCT_POLICY := android.policy_phone
 
 PRODUCT_PACKAGES := \
-    DeskClock \
     Bluetooth \
     Calculator \
     Calendar \
-    Camera2 \
     CertInstaller \
-    Email \
-    Exchange2 \
     FusedLocation \
-    Gallery2 \
     InputDevices \
     Keyguard \
     LatinIME \
     Launcher2 \
-    Music \
     OneTimeInitializer \
     PrintSpooler \
     Provision \
diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/languages_full.mk cm11/build/target/product/languages_full.mk
--- cm11-backup/build/target/product/languages_full.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/languages_full.mk	2014-11-14 05:18:36.244875224 +0100
@@ -21,7 +21,4 @@
 
 # These are all the locales that have translations and are displayable
 # by TextView in this branch.
-PRODUCT_LOCALES := en_US en_IN fr_FR it_IT es_ES et_EE de_DE nl_NL cs_CZ pl_PL ja_JP zh_TW zh_CN zh_HK ru_RU ko_KR nb_NO es_US da_DK el_GR tr_TR pt_PT pt_BR rm_CH sv_SE bg_BG ca_ES en_GB fi_FI hi_IN hr_HR hu_HU in_ID iw_IL lt_LT lv_LV ro_RO sk_SK sl_SI sr_RS uk_UA vi_VN tl_PH ar_EG fa_IR th_TH sw_TZ ms_MY af_ZA zu_ZA am_ET en_XA ar_XB fr_CA km_KH lo_LA ne_NP si_LK mn_MN hy_AM az_AZ ka_GE
-
-# CyanogenMod
-PRODUCT_LOCALES += es_XA lb_LU ku_IQ ug_CN
+PRODUCT_LOCALES := en_US en_GB fr_FR it_IT de_DE es_ES
diff -U 3 -H -d -r -N -- cm11-backup/build/target/product/locales_full.mk cm11/build/target/product/locales_full.mk
--- cm11-backup/build/target/product/locales_full.mk	2014-11-13 22:50:20.000000000 +0100
+++ cm11/build/target/product/locales_full.mk	2014-11-14 05:18:58.770873647 +0100
@@ -1,6 +1,3 @@
-PRODUCT_LOCALES := en_US cs_CZ da_DK de_AT de_CH de_DE de_LI el_GR en_AU en_CA en_GB en_NZ en_SG eo_EU es_ES fr_CA fr_CH fr_BE fr_FR it_CH it_IT ja_JP ko_KR nb_NO nl_BE nl_NL pl_PL pt_PT ru_RU sv_SE tr_TR zh_CN zh_HK zh_TW am_ET hi_IN
-
-# CyanogenMod
-PRODUCT_LOCALES += es_XA lb_LU ku_IQ ug_CN
+PRODUCT_LOCALES := en_US en_GB fr_FR it_IT de_DE es_ES
 
 $(call inherit-product, build/target/product/languages_full.mk)
